addon/data/shoot-panel.js

```diff
@@ -59,20 +65,24 @@ let ShootPanel = React.createClass({
         <a href="#">+</a>
       </div>
       <div className="link-row">
-        <a className="link" target="_blank" href={ this.props.viewUrl } onClick={ this.onLinkClick }>{ this.props.viewUrl }</a>
+        <a className="link" target="_blank" href={ this.props.shot.viewUrl } onClick={ this.onLinkClick }>{ this.props.shot.viewUrl }</a>
         <button className="copy" ref="copy" type="button" data-normal-text="Copy Link" data-copied-text="Copied!" onClick={ this.onCopyClick }>Copy Link</button>
       </div>
 
 
-      <div className="text">{ this.props.textSelection }</div>
-      <div className="comment">{ this.props.comment}</div>
+      <div className="text">{ this.props.shot.textSelection }</div>
+      <div className="comment">{ this.props.shot.comment}</div>
       <input className="comment-input" ref="input" type="text" placeholder="Say something" onKeyup={ this.onKeyup }/>
     </div>;
   }
 });
 
 self.port.on("shotData", err.watchFunction(function (data) {
-  React.render(React.createElement(ShootPanel, data), document.body);
+  console.log("shotData", Object.getOwnPropertyNames(data));
+  let as = new shot.AbstractShot(data.backend, data.id, data.shot);
```

Comment:
Should just be called a shot – I'm guessing we'll want to subclass AbstractShot for the panel eventually, but either way the instance isn't abstract